Delegate_CategorySampleApp/CustomAlertView.h

```diff
@@ -0,0 +1,28 @@
+//
+//  CustomAlertView.h
+//  Delegate_CategorySampleApp
+//
+//  Created by webonise on 21/09/16.
+//  Copyright © 2016 webonise. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+@protocol CustomAlertViewDelegate <NSObject>
+-(void) onButton1Pressed;
+-(void) onButton2Pressed;
+@end
+
+@interface CustomAlertView : UIView
+@property(nonatomic) id <CustomAlertViewDelegate> delegateProperty;
+//-(void) designAlertView :(NSString*)labelButtonMessage :(NSString*)titleButton1 :(NSString*)titleButton2;
+@property(strong, nonatomic) NSString* labelAlertViewMessage;
+@property(strong, nonatomic) NSString* buttonOneText;
+@property(strong, nonatomic) NSString* buttonTwoText;
+-(void) designAlertView;
```

Comment:
no need to take 
NSString* labelAlertViewMessage;
NSString* buttonOneText;
NSString* buttonTwoText;
these as public property make them local to method.